docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1468,3 +1468,136 @@ This can be done in using startup script. See [cloudInitNoCloud](#cloudInitNoClo
 - Windows Example
 
    See [this](https://virtio-fs.gitlab.io/howto-windows.html) guide for details on startup steps needed for Windows VMs.
+
+### Sharing Node Directories with VMs
+
+`Virtiofs` allows to share node directories with VMs as [hostpaths](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
```

Comment:
IMO, this sentence is a bit misleading. Virtiofs allows sharing directories directly with the guest, not only host paths.  The directory is provided through a PVC, and this can be backed by a hostpath

Reply:
I agree. I have changed it.